edsnlp/pipes/__init__.py

```diff
@@ -73,9 +76,13 @@
     from .qualifiers.negation.factory import create_component as negation
     from .qualifiers.reported_speech.factory import create_component as reported_speech
     from .qualifiers.reported_speech.factory import create_component as rspeech
+    from .trainable.embeddings.span_pooler.factory import (
+        create_component as span_pooler,
+    )
+    from .trainable.embeddings.text_cnn.factory import create_component as text_cnn
+    from .trainable.embeddings.transformer.factory import (
+        create_component as transformer,
+    )
     from .trainable.ner_crf.factory import create_component as ner_crf
     from .trainable.span_classifier.factory import create_component as span_classifier
     from .trainable.span_linker.factory import create_component as span_linker
-    from .trainable.embeddings.span_pooler.factory import create_component as span_pooler
-    from .trainable.embeddings.transformer.factory import create_component as transformer
-    from .trainable.embeddings.text_cnn.factory import create_component as text_cnn
```

Comment:
You should avoid reformating this file (i guess this is your IDE settings since this file is marked as ignorable in the pyproject ruff settings) and only add your pipe to the list